src/bench/base58.cpp

```diff
@@ -13,7 +13,7 @@
 
 static void Base58Encode(benchmark::State& state)
 {
-    unsigned char buff[32] = {
```

Comment:
@sipa To get rid of `clang-tidy`, Clang Static Analyzer and other static analyser warnings along the lines of:
```c++
bench/base58.cpp:23:9: warning: Function call argument is a pointer to uninitialized value [clang-analyzer-core.CallAndMessage]
```